src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala

```diff
@@ -33,6 +33,9 @@ import scala.collection.immutable._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
+object JdbcAsyncWriteJournal {
+  val Identifier = "jdbc-journal"
```

Comment:
final val provides byte code optimization for these kinds of identifiers.  Try it out in the REPL and look at the 
resulting byte code eg: 

```
object A { val X = "Hello" }
:javap -filter -c A
```

vs

```
object A { final val X = "Hello" }
:javap -filter -c 
```

Also calls to these identifiers (final val) are inlined in the call graph eg:

```
scala> object A { final val X = "Hello" }
scala> object B { final val Z = A.X }
scala> :javap -filter -c B
```